samples/nrf9160/lwm2m_carrier/src/main.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <lwm2m_carrier.h>
+#include <zephyr.h>
+
+void bsd_recoverable_error_handler(uint32_t error)
+{
+	printk("RECOVERABLE ERROR %lu\n", (unsigned long)error);
```

Comment:
All examples should be aligned. Printing an unsigned `%u` value should be enough here.

Reply:
Done.